api/lib/ZwaveClient.ts

```diff
@@ -4133,7 +4418,11 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 		rounds = 5,
 	): Promise<LifelineHealthCheckSummary & { targetNodeId: number }> {
 		if (this.driverReady) {
-			const result = await this.getNode(nodeId).checkLifelineHealth(
+			const zwaveNode = this.getNode(nodeId)
+			if (!zwaveNode || nodeId > 0xfff) {
+				throw new Error(`Node ${nodeId} not found or is a virtual node`)
+			}
```

Comment:
add a paramenter on getNode named `options` with only a property named `noVirtual` that by default is false and when true throws this error, reuse this where needed